packages/core/src/runtime/getStyleBucketName.ts

```diff
@@ -52,20 +57,22 @@ export function getStyleBucketName(pseudo: string, layer: string, media: string,
     return 't';
   }
 
-  const normalizedPseudo = pseudo.trim();
+  if (selectors.length > 0) {
+    const normalizedPseudo = selectors[0].trim();
```

Comment:
We take into account only the first pseudo selector, so it's find to use the first element.